src/GafferArnold/ArnoldLight.cpp

```diff
@@ -115,3 +115,17 @@ IECoreScene::ConstShaderNetworkPtr ArnoldLight::computeLight( const Gaffer::Cont
 	IECore::ConstCompoundObjectPtr shaderAttributes = shaderInPlug()->attributes();
 	return shaderAttributes->member<const IECoreScene::ShaderNetwork>( "ai:light" );
 }
+
+void ArnoldLight::hashStandardSetNames( const Gaffer::Context *context, IECore::MurmurHash &h ) const
+{
+	h.append( namePlug()->getValue() );
```

Comment:
This should first call the base class `Light::hashStandardSetNames()`, to account for the base class behaviour.